BUILD.gn

```diff
@@ -191,7 +152,7 @@ static_library("libdeno") {
   # header file that contains the the sha256 hash of the snapshot.
   if (cc_wrapper != "" && cc_wrapper != "ccache") {
     hash_h = "$target_gen_dir/bundle/hash.h"
-    inputs += [ hash_h ]
+    data += [ hash_h ]
```

Comment:
This is not right -- the hash is not a data dep.
It's a build-time dependency of the target that embeds the snapshot.

Reply:
Fixed.